### ISSUE TITLE
Does this plugin work with the 1.9.0 release candidates

### ISSUE THREAD

**Issue by b-morgan** [poulou0/OctoPrint-WebcamExtras#9]
There are some changes in the camera support in the [https://community.octoprint.org/t/new-release-candidate-1-9-0rc5/51196](https://community.octoprint.org/t/new-release-candidate-1-9-0rc5/51196). Does this plugin work with 1.9.0?

**Comment by poulou0** [comment]
rc5 should be good as far as I checked.
https://github.com/OctoPrint/plugins.octoprint.org/pull/1182#issuecomment-1518754306

But there is always room for a bug! 😊
Did you try and it didn't work for you?

**Comment by b-morgan** [comment]
I didn't try it. My printer is attached to an RPi 3B (with an RPi camera V2) running OctoPi 0.18.0 with OctoPrint 1.8.7. I tested with one of the earlier 1.9.0 release candidates and had some issues with other plugins so I need to backup the current image and try again. I'll add this plugin when I do that.

**Comment by poulou0** [comment]
Awesome! Let me know how it goes and we can iron things out if it doesn't.

BTW if you are using it any star in github will be appreciated ⭐

**Comment by b-morgan** [comment]
I have now tried this on the new camera stack image and I had a problem. On 1.8.7, the hflip and vflip checkboxes are on the settings, Webcam & Timelapse tab. On 1.9.0rc5, they have moved to the Classic Webcam plugin settings.

I installed the plugin on 1.8.7, played with the checkboxes and I believe I finished with them all unchecked. I installed 1.9.0rc5 and the camera view was flipped horizonally. At that time I didn't know where the checkboxes had gone so I reimaged the RPi, installed 1.9.0rc5, and then installed this plugin. The camera image is now correct and the checkboxes appear to work correctly.
